test/spec/token.js

```diff
@@ -32,6 +32,38 @@ describe('token.decode', function () {
 });
 
 describe('token.getWithoutPrompt', function () {
+  it('uses defaults if undefined/null values are provided for requesting an id_token using a sessionToken', function(done) {
```

Comment:
In this test description, it might be clearer if you explicitly mention that you care about the query params that get built, given the configuration input.